Dockerfile

```diff
@@ -1,15 +1,18 @@
+ARG tomcat_version=9.0.29
 
 FROM alpine
 
+ARG tomcat_version
+
 LABEL maintainer="devopsa3"
 
 RUN apk add openjdk8 curl
 
 RUN cd /usr/local/ \
-  && wget http://apache.volia.net/tomcat/tomcat-8/v8.5.47/bin/apache-tomcat-8.5.47.tar.gz \
-  && tar xzf apache-tomcat-8.5.47.tar.gz \
-  && mv apache-tomcat-8.5.47/ tomcat/ \
-  && rm apache-tomcat-8.5.47.tar.gz
+  && wget http://apache.ip-connect.vn.ua/tomcat/tomcat-9/v${tomcat_version}/bin/apache-tomcat-${tomcat_version}.tar.gz \
```

Comment:
url is dependent on major version,
image won't be built with `ARG tomcat_version=8.0.0 `

Reply:
Fixed